central/src/containers/RouteContainer.tsx

```diff
@@ -127,7 +150,26 @@ export const RouteContainer = ({
     }  
     setNextToken((prev) => nextToken);
   }
-  
+
```

Comment:
Whenever a user updates the search text field, the onChange is debounced by .5s to prevent continuous API calls. `filterString` is used to receive only templates that match the search bar input